assignments/serializers.py

```diff
@@ -28,16 +30,86 @@ class BudgetingTaskSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = BudgetingTask
-        fields = ['name', 'unit', 'targets', 'amount_of_consumption']
+        fields = ['id', 'name', 'unit', 'targets', 'amount_of_consumption']
+        extra_kwargs = {
```

Comment:
DRF has a `read_only_field` option that can be passed to the Meta class:
http://www.django-rest-framework.org/api-guide/serializers/#specifying-read-only-fields